alignak/log.py

```diff
@@ -294,14 +294,15 @@ def error(self, *args, **kwargs):
 
 # --- create the main logger ---
 logging.setLoggerClass(Log)
+# pylint: disable=C0103
 logger = logging.getLogger('Alignak')
 if hasattr(sys.stdout, 'isatty'):
-    csh = ColorStreamHandler(sys.stdout)
+    CSH = ColorStreamHandler(sys.stdout)
```

Comment:
Still a constant because defined in module directly. Actually we should change all this but it was not a simple one